homeassistant/components/kef/media_player.py

```diff
@@ -85,6 +85,16 @@
 )
 
 
+def get_ip_mode(host):
```

Comment:
What was the reason for this change? This function is only used in one place so seems redundant.